testdata/ci/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/DATA-DOG/godog v0.7.13
 	github.com/PuerkitoBio/goquery v1.5.1
-	github.com/alien4cloud/alien4cloud-go-client/v2 v2.2.0-milestone.6
+	github.com/alien4cloud/alien4cloud-go-client/v2 v2.2.0-milestone.12.0.20201011203603-8baf080a1357
```

Comment:
maybe we should wait for a milestone before merging this PR

Reply:
Exactly, I'm waiting for a review of this PR: https://github.com/alien4cloud/alien4cloud-go-client/pull/34 :-)
This is why I didn't set you yet as a reviewer here. I just prepare the PR to run continuous integration on it.
I should have tag it as draft in addition